corehq/apps/domain/views/settings.py

```diff
@@ -540,7 +541,11 @@ def form(self):
 
     def post(self, request, *args, **kwargs):
         if self.form.is_valid():
-            # ToDo: save the alert
+            CommCareHQAlert.objects.create(
+                created_by_domain=self.domain,
+                domains=[self.domain],
```

Comment:
This is necessary despite setting created_by_domain to let HQ know the relevant domain for the alert during filtering for UI.